app/src/main/java/io/gnosis/safe/utils/NavUtils.kt

```diff
@@ -0,0 +1,14 @@
+package io.gnosis.safe.utils
+
+import androidx.fragment.app.FragmentManager
+import androidx.navigation.NavDirections
+import androidx.navigation.fragment.NavHostFragment
+
+fun navigateFromChild(childFragmentManager: FragmentManager, directions: NavDirections) {
+    try {
+        (childFragmentManager.primaryNavigationFragment as NavHostFragment).navController.navigate(directions)
```

Comment:
Maybe write this as an extension `fun FragmentManager.getChildNavController(): NavController?`. Also it's probably better to use a safe cast `as?` and if you still need the try and catch please put the exception to `Timber`